scripts/cli/lec.sh

```diff
@@ -513,6 +532,27 @@ cmd_remove() {
 
 	_print_success "Project ${worktree_name} removed"
 }
+cmd_share(){
+	local exportFlag="${1}"
+
+	_checkCWDProject
+
+	_print_step "Zipping up workspace..."
+
+	(
+		cd "${CWD_PROJECT_ROOT}" || exit
+
+		if ! ./gradlew shareWorkspace; then
+			exit 1
+		fi
+
+		if [[ "${exportFlag}" == "--export" ]]; then
+			cmd_exportData
+		fi
```

Comment:
This is out of order. The data export needs to happen before the `shareWorkspace` Gradle task is run.